app/utils/API1811/ChargeTypes.scala

```diff
@@ -80,59 +125,44 @@ object ChargeTypes extends LoggerUtil {
-    ("4666", vatInterestSubTrans) -> "VAT Error Correct 1st LPP LPI",
-    ("4668", vatInterestSubTrans) -> "VAT Error Correct 2nd LPP LPI",
-    ("4670", vatInterestSubTrans) -> "VAT Additional Assessment LPI",
-    ("4672", vatInterestSubTrans) -> "VAT AA 1st LPP LPI",
-    ("4674", vatInterestSubTrans) -> "VAT AA 2nd LPP LPI",
-    ("4676", vatInterestSubTrans) -> "VAT Protective Assessment LPI",
-    ("4678", vatInterestSubTrans) -> "VAT PA 1st LPP LPI",
-    ("4680", vatInterestSubTrans) -> "VAT PA 2nd LPP LPI",
-    ("4686", vatInterestSubTrans) -> "VAT POA Instalment LPI",
-    ("4689", vatInterestSubTrans) -> "VAT AA Quarterly Instal LPI",
-    ("4691", vatInterestSubTrans) -> "VAT AA Monthly Instal LPI",
-    ("4703", penaltyDebitSubTrans) -> "VAT Return 1st LPP",
-    ("4704", penaltyDebitSubTrans) -> "VAT Return 2nd LPP",
-    ("4716", penaltyDebitSubTrans) -> "VAT POA Return 1st LPP",
-    ("4717", penaltyDebitSubTrans) -> "VAT POA Return 2nd LPP",
-    ("4718", penaltyDebitSubTrans) -> "VAT AA Return Charge 1st LPP",
-    ("4719", penaltyDebitSubTrans) -> "VAT AA Return Charge 2nd LPP",
-    ("4723", penaltyDebitSubTrans) -> "VAT Central Assessment 1st LPP",
-    ("4724", penaltyDebitSubTrans) -> "VAT Central Assessment 2nd LPP",
-    ("4741", penaltyDebitSubTrans) -> "VAT OA 1st LPP",
-    ("4742", penaltyDebitSubTrans) -> "VAT OA 2nd LPP",
-    ("4743", penaltyDebitSubTrans) -> "VAT Error Correction 1st LPP",
-    ("4744", penaltyDebitSubTrans) -> "VAT Error Correction 2nd LPP",
-    ("4748", penaltyDebitSubTrans) -> "VAT Late Submission Pen",
-    ("4749", vatInterestSubTrans) -> "VAT LSP Interest",
-    ("4758", penaltyDebitSubTrans) -> "VAT AA 1st LPP",
-    ("4759", penaltyDebitSubTrans) -> "VAT AA 2nd LPP",
-    ("4761", penaltyDebitSubTrans) -> "VAT PA 1st LPP",
-    ("4762", penaltyDebitSubTrans) -> "VAT PA 2nd LPP",
-    ("4787", penaltyDebitSubTrans) -> "VAT Manual LPP",
-    ("4788", vatInterestSubTrans) -> "VAT Manual LPP LPI"
+    ("7710", vatInterestRepaymentsSubTrans) -> "VAT Repayment Supplement",
+    ("7720", vatDebitSubTrans) -> "VAT Central Assessment",
+    ("7721", vatInterestSubTrans) -> "VAT Default Interest",
+    ("7722", vatInterestSubTrans) -> "VAT Further Interest",
+    ("7730", vatDebitSubTrans) -> "VAT OA Debit Charge",
+    ("7730", vatCreditSubTrans) -> "VAT OA Credit Charge",
+    ("7731", vatDebitSubTrans) -> "VAT EC Debit Charge",
+    ("7731", vatCreditSubTrans) -> "VAT EC Credit Charge",
+    ("7735", penaltyDebitSubTrans) -> "VAT Miscellaneous Penalty",
+    ("7745", penaltyDebitSubTrans) -> "VAT Civil Evasion Penalty",
+    ("7747", penaltyDebitSubTrans) -> "VAT Debit Default Surcharge",
+    ("7755", penaltyDebitSubTrans) -> "VAT Inaccuracy Assessments pen",
+    ("7760", penaltyDebitSubTrans) -> "VAT BNP of Reg Post 2010",
+    ("7765", penaltyDebitSubTrans) -> "VAT Wrong Doing Penalty",
+    ("7766", penaltyDebitSubTrans) -> "VAT FTN mat change post 2010",
```

Comment:
The String here doesn't match the non-mig version exactly. The frontend may convert these to all lower/upper case anyway, I don't remember, but regardless it's probably best to match them identically here